docs/api_reference/index.rst

```diff
@@ -13,3 +13,108 @@ API Docs
    fairlearn.reductions
    fairlearn.adversarial
    fairlearn.experimental
```

Comment:
This is exactly what I was thinking of, excellent! I am wondering if we need the module-specific pages, i.e., everything above this line (`fairlearn.*`) as separate pages at all. After all, we have all their contents below.

Wdyt?

Reply:
I don't have a preference, it would be a pretty easy change - but I suppose we don't need them since readers can just use the side menu to navigate to a page for a specific module?

Reply:
I guess I'm wondering if we should remove the module-specific pages altogether, so that they wouldn't even show up in the side menu. They aren't really useful unless they have additional content. AFAICT that's what scikit-learn does as well. There's only https://scikit-learn.org/stable/modules/classes.html#module-sklearn.covariance but no sub-pages per module. They do link relevant user guide sections in the section description for each module (if any).

Reply:
>I guess I'm wondering if we should remove the module-specific pages altogether, so that they wouldn't even show up in the side menu. 

I'm not sure if I fully understand your suggestion @romanlutz. Scikit-learn has module-level navigation on the side menu which directs to the specific location in the (single page) API reference overview. So while they don't have module specific pages, the side menu does have module level navigation. Are you suggesting no module-level navigation at all or just no module-level pages?

Reply:
Correct. I'm just saying let's not have module-specific pages unless we actually put useful content there (which we currently don't have). Otherwise it's just one additional click. So as long as the module-specific links just take you to the section on the same page that's fine, but I had understood the PR differently. I should probably run this locally and see what it looks like, but I don't have time before next week...